domains/hatimzafr.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "hatimzafr",
+        "email": "hatimfatih1504@gmail.com",
+        "discord": ""
```

Why put discord if you arent going to put anything in it?
```suggestion
        "email": "hatimfatih1504@gmail.com"
```